news/2024/2024-03-03-4dmwc-2024-concludes.md

```diff
@@ -0,0 +1,171 @@
+
+## Highlights
+
+As it is to be expected, a tournament of this caliber had a bunch of moments worth putting under the spotlight! And what better way to do that than to recap everything into a video. Thanks to [Polytetral](https://osu.ppy.sh/users/8612061) and the vast amount of submissions they got, we'd like to present to you a recap of the tournament's highlights.
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/cXkiX7u4a9g" frameborder="0" allowfullscreen></iframe>
+</div>
+
+From commentary bloopers, to close calls, to nerve-wracking moments all across the tournament. The video has it all! For those who want to get a taste of the best (and funniest) clips, feel free to watch the video. Alternatively if you want the long version of events, all VODs can be found in the [4digitmwc Twitch channel](https://www.twitch.tv/collections/2LFhgDzhpxcVmQ).
+
+## osu! originals
+
+For many users, this is one of a tournament's biggest highlights. A labour of love, and an original art piece through and through in combination with its accompanying beatmap. Many thanks to [Akiri](https://osu.ppy.sh/users/19587411), [BrayanKitsn](https://osu.ppy.sh/users/34744443), and [SiLiS](https://soundcloud.com/silis) (AKA [Raveille](https://osu.ppy.sh/users/1388767)) for their involvement in the original tracks!
+
+### Round of 32
+
+[![](/wiki/shared/news/2024-03-03-4dmwc-2024-concludes/plana-stella-achernar.jpg)](https://osu.ppy.sh/beatmapsets/2119559#mania/4452403)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://wala.s-ul.eu/HuEppZpP.mp3">
+    </audio>
+</div>
+
+### Round of 16
+
+[![](/wiki/shared/news/2024-03-03-4dmwc-2024-concludes/game-make-over.jpg)](https://osu.ppy.sh/beatmapsets/2123356#mania/4462335)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://wala.s-ul.eu/YUZ9VLD3.mp3">
+    </audio>
+</div>
+
+### Finals
+
+[![](/wiki/shared/news/2024-03-03-4dmwc-2024-concludes/beyond-wood.jpg)](https://osu.ppy.sh/beatmapsets/2134266#mania/4490359)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://wala.s-ul.eu/8iz3aFnU.mp3">
+    </audio>
+</div>
+
+## Interviews
+
+Getting to know the thoughts of those most closely involved with the project is a good chance to reflect on some user experiences throughout the tournament. Not only does it give them the chance to introspect their contributions, but it gives us — the readers — a perspective and insight that we would've otherwise had missed!
```

Comment:
having two en dashes feels too disruptive to me
```suggestion
Getting to know the thoughts of those most closely involved with the project is a good chance to reflect on some user experiences throughout the tournament. Not only does it give them the chance to introspect their contributions, but it gives us — the readers, a perspective and insight that we would've otherwise had missed!
```

Reply:
that's how em dashes are supposed to be used (enclosing an interjection), much like parentheses

Reply:
Assuming it's A-Okay then